clarity/src/vm/functions/define.rs

```diff
@@ -152,6 +152,7 @@ fn handle_define_function(
         define_type,
         function_name,
         &env.contract_context.contract_identifier.to_string(),
+        TypeSignature::NoType, // FIXME: this is a placeholder
```

Comment:
Is this fixable before merge, or will this be remembered? :)

Reply:
This new return type field is only used in the new code paths, for Wasm. So the question is... should we just leave it as a `NoType` in this interpreter path, or should we make it an `Option` and set it to `None`, or should we go through the trouble of setting it in the interpreter path? Any thoughts?

Reply:
I'm personally not a huge fan of having different meanings for fields dependent on code path... so if it were me I would probably set it in all paths if it would make sense... but if that's a bunch of work, the next best option imo would be to use the `Option+None` variant and a comment on the field indicating when it can be `Some`.

Reply:
I think setting it here is a bunch of work that doesn't give us any benefits. I'll update this to use an `Option<TypeSignature>`.

Reply:
Updated in 164b9299e145d5bceafcbc993095b3552069c661